modules/dicom_archive/php/viewdetails.class.inc

```diff
@@ -255,16 +250,13 @@ class ViewDetails extends \NDB_Form
         $sliceThickness = $series['SliceThickness'];
 
         foreach ($this->protocols as $protocol) {
-            if ((! $protocol['TR_range']
-                || $this->_inRange($tr, $protocol['TR_range']))
-                && (! $protocol['TE_range']
-                || $this->_inRange($te, $protocol['TE_range']))
-                && (! $protocol['TI_range']
-                || $this->_inRange($ti, $protocol['TI_range']))
-                && (! $protocol['slice_thickness_range']
-                || $this->_inRange(
+            if (($this->_inRange($tr, $protocol['TR_min'], $protocol['TR_max']))
```

Comment:
the logic here is not working. It will return unknown for all protocols that do not have `TR`, `TE`, `TI` AND `slice thickness` set. However, many acquisitions do not even have a `TI`. In the screenshot attached, the only scan type I have with all those values set are the t1 scans. Everything else would fail identifying the scan type.

![Screen Shot 2019-03-11 at 3 38 35 PM](https://user-images.githubusercontent.com/1402456/54152690-3625ea80-4414-11e9-9378-aefa998eabb4.png)

Reply:
@llevitis @cmadjar is this resolved?

Reply:
yes, which is why I approved the PR ;).